### PR TITLE
fix: Fix Main window is not set yet exception on Skia

### LINKED ISSUE #12737: Skia crashes with `System.InvalidOperationException: 'Main window is not set yet.'` with 5.0.0-dev.1155
### Current behavior

image

### Expected behavior

_No response_

### How to reproduce it (as minimally and precisely as possible)

_No response_

### Workaround

_No response_

### Works on UWP/WinUI

None

### Environment

_No response_

### NuGet package version(s)

_No response_

### Affected platforms

_No response_

### IDE

_No response_

### IDE version

_No response_

### Relevant plugins

_No response_

### Anything else we need to know?

_No response_

### PR DESCRIPTION
**Pull request by Youssef1313**
GitHub Issue (If applicable): closes #12737

## PR Type

What kind of change does this PR introduce?

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

File: src/Uno.UI.Runtime.Skia.Gtk/UI/Controls/UnoGtkWindow.cs [hunk 1]
```diff
@@ -52,9 +52,6 @@ public UnoGtkWindow(WinUIWindow winUIWindow) : base(WindowType.Toplevel)
 
 		Host = new UnoGtkWindowHost(this, winUIWindow);
 
-		UpdateWindowPropertiesFromPackage();
```
Comment (Youssef1313) [inline src/Uno.UI.Runtime.Skia.Gtk/UI/Controls/UnoGtkWindow.cs:55]: This was called from `UnoGtkWindow` constructor, but down the stack, it is expecting the window to be non-null, but it hasn't been yet created. So I moved these after the constructor finishes execution and `MainWindow` is assigned a non-null value.

**Review by jeromelaban — APPROVED** [review]

**Review by agneszitte — APPROVED** [review]

**Review by Youssef1313 — COMMENTED** [review]
@MartinZikmund Can you help with the test failures?

I think it might be the case that `OnShown` is called later than what the test is expecting. Should I move out of `OnShown` to the line next to the constructor call as I was previously doing? Or should the test be adjusted somehow?

**Comment by jeromelaban** [comment]
This test is failing: https://dev.azure.com/uno-platform/Uno%20Platform/_build/results?buildId=84303&view=ms.vss-test-web.build-test-results-tab&runId=2412545&resultId=100388&paneView=debug

**Comment by Youssef1313** [comment]
I reverted the change to move the code to `OnShown` to see if the test is going to pass now.

File: src/Uno.UI.Runtime.Skia.Gtk/UI/Controls/UnoGtkWindow.cs (hunk 1 above)
Comment (Youssef1313) [reply src/Uno.UI.Runtime.Skia.Gtk/UI/Controls/UnoGtkWindow.cs:55]: I reverted the `OnShown` change due to test failure.

**Pull request merged**